.github/workflows/update-swagger.yml

```diff
@@ -47,13 +49,15 @@ jobs:
           # Set git name, mail and origin
           git config --global user.name "${{ secrets.GREENBONE_BOT }}"
           git config --global user.email "${{ secrets.GREENBONE_BOT_MAIL }}"
```

Comment:
btw. we have an action for this ... 🤷🏿‍♂️ 
(https://github.com/greenbone/actions/tree/main/set-github-user)